app/migration/migrators/school.rb

```diff
@@ -67,6 +99,13 @@ def find_gias_school(urn:) = gias_schools.find { |school| school.urn == urn }
 
     def gias_schools = @gias_schools ||= ::GIAS::School.where(urn: self.class.schools.pluck(:urn).sort)
 
+    def migrate_school!(ecf_school)
+      return false if ecf_school.open?
+      return false unless ecf_school.with_induction_records?
```

Comment:
Something like `return false if ecf_school.induction_records.blank?` feels more intuitive to me, and we could drop the `with_induction_records?` method from the model.

Reply:
Perfect. I created it before `Migration::School#inductions` was defined. 
Now that we have the associations it's easier to call it and remove the method. 
Thanks.